x/data/client/testsuite/tx.go

```diff
@@ -0,0 +1,175 @@
+package testsuite
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktestutil "github.com/cosmos/cosmos-sdk/x/bank/client/testutil"
+	"github.com/regen-network/regen-ledger/types/testutil/cli"
+	"github.com/regen-network/regen-ledger/types/testutil/network"
+	"github.com/regen-network/regen-ledger/x/data/client"
+	"github.com/stretchr/testify/suite"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg     network.Config
+	network *network.Network
+}
+
+func NewIntegrationTestSuite(cfg network.Config) *IntegrationTestSuite {
+	return &IntegrationTestSuite{cfg: cfg}
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+
+	// We execute NewIntegrationTestSuite to set cfg field of IntegrationTestSuite
```

Comment:
`NewIntegrationTestSuite` is not called within this block. Is this comment still relevant?

Reply:
ah good catch, removed it. looks like that comment is still in the group tx.go file (where i copied from), will remove that as well